sdks/java/nexmark/src/main/java/org/apache/beam/sdk/nexmark/NexmarkLauncher.java

```diff
@@ -860,14 +866,14 @@ private void sinkEventsToPubsub(PCollection<Event> events, long now) {
    * Send {@code formattedResults} to Kafka.
    */
   private void sinkResultsToKafka(PCollection<String> formattedResults) {
-    checkArgument(!Strings.isNullOrEmpty(options.getBootstrapServers()),
-            "Missing --bootstrapServers");
+    NexmarkUtils.console("Writing results to Kafka Topic %s", options.getKafkaResultsTopic());
```

Comment:
Move this line after `checkArgument` in next line.

Reply:
Done in all places.